lib/imports/emails.js

```diff
@@ -67,11 +96,15 @@ export const registrationInfoHTML = `
 </p>
 `;
 
-export const questions = `<p>If you have any questions please email <a href="mailto:support@greatpuzzleHunt.com">support@greatpuzzlehunt.com</a></p>`;
+export const questions = `<p>If you have any questions please email <a href="${supportEmail}">${supportEmail}</a></p>`;
```

Comment:
Should this be `href="mailto:${supportEmail}"`?

Reply:
Fixed